src/css/app.css

```diff
@@ -7,18 +7,42 @@
 .navbar {
```

Comment:
we named this as **App.css** with capital A, in order to avoid conflicts. so this should be renamed as well

Reply:
i am going to work on it bro